docs/containers/reference.md

```diff
@@ -268,3 +268,147 @@ Here are all properties available for configuring `docker-run` task. All propert
 | `enableDebugging` |  Whether or not to enable debugging within the container. | `false` |
 | `inspectMode` |  Defines the initial interaction between the application and the debugger (`default` or `break`). <br/> The value `default` allows the application to run until the debugger attaches. <br/> The value `break` prevents the application from running until the debugger attaches. | `default` |
 | `inspectPort` |  The port on which debugging should occur. | `9229` |
+
+## Command customization
+
+The Docker extension executes a number of Docker CLI commands on behalf of its users, such as to build images, run containers, attach to containers, and view container logs. Some of these commands have a large number of optional arguments, often used in very specific scenarios. Many of these commands can be customized.
```

Comment:
"on behalf of its users" I think can be improved.  Can this be more exact?  Something like "when you perform various operations, such as to build images, run containers ..."